### PR TITLE
refactor: initial protos + skeleton for timeline service

### PR DESCRIPTION
**Pull request by worstell**
pulled shared protos from `console.proto` into new `event.proto`

### CODE REVIEW

**Review by matt2e — APPROVED** [review]

**Review by alecthomas — APPROVED** [review]
I think @stuartwdouglas has started moving service .proto files to their own directories, so we should stay consistent with that.

File: backend/timeline/service.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+	ftlv1 "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1"
+	"github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/ftlv1connect"
+	"github.com/TBD54566975/ftl/internal/log"
+	"github.com/TBD54566975/ftl/internal/rpc"
+	"github.com/TBD54566975/ftl/internal/schema/schemaeventsource"
+)
+
+type Config struct {
+	Bind *url.URL `help:"Socket to bind to." default:"http://127.0.0.1:8894" env:"FTL_TIMELINE_BIND"`
```
Comment (alecthomas) [inline backend/timeline/service.go:19]: Use FTL_BIND

File: cmd/ftl-timeline/main.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	"github.com/TBD54566975/ftl/internal/schema/schemaeventsource"
+)
+
+var cli struct {
+	Version             kong.VersionFlag     `help:"Show version."`
+	ObservabilityConfig observability.Config `embed:"" prefix:"o11y-"`
+	LogConfig           log.Config           `embed:"" prefix:"log-"`
+	TimelineConfig      timeline.Config      `embed:"" prefix:"timeline-"`
+	ConfigFlag          string               `name:"config" short:"C" help:"Path to FTL project cf file." env:"FTL_CONFIG" placeholder:"FILE"`
+	ControllerEndpoint  *url.URL             `name:"ftl-endpoint" help:"Controller endpoint." env:"FTL_ENDPOINT" default:"http://127.0.0.1:8892"`
```
Comment (alecthomas) [inline cmd/ftl-timeline/main.go:29]:
```suggestion
	SchemaServiceEndpoint  *url.URL             `name:"ftl-endpoint" help:"SchemaService endpoint." env:"FTL_ENDPOINT" default:"http://127.0.0.1:8892"`
```

File: cmd/ftl-timeline/main.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+func main() {
+	t, err := strconv.ParseInt(ftl.Timestamp, 10, 64)
+	if err != nil {
+		panic(fmt.Sprintf("invalid timestamp %q: %s", ftl.Timestamp, err))
+	}
+	kctx := kong.Parse(&cli,
+		kong.Description(`FTL - HTTP Ingress`),
```
Comment (alecthomas) [inline cmd/ftl-timeline/main.go:38]:
```suggestion
		kong.Description(`FTL - Timeline`),
```

**Pull request merged**